src/main/webapp/less/portlet-grid.less

```diff
@@ -0,0 +1,142 @@
+/**
+ * Licensed to Apereo under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work
+ * for additional information regarding copyright ownership.
+ * Apereo licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License.  You may obtain a
+ * copy of the License at the following location:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*---------------------------------------------------
+    PORTLET GRID Mixin   | version: 0.7
+  ---------------------------------------------------
+    A mixin to reflow Bootstrap Classes and to create
```

Comment:
okay, that makes more sense.
I take it this is why you are interested in [`@element` queries](http://elementqueries.com/)?